### PR TITLE
chore(agent): add missing setValue() docs

### PR DESCRIPTION
**Pull request by polaroi8d**
Add missing `setValue()` `godocs`, please @nandor-magyar review.

### CODE REVIEW

File: golang/internal/config/config.go [hunk 1]
```diff
@@ -4,11 +4,18 @@ import (
 	"time"
 )
 
-// configuration defaults
-// convention is we have ALL the options here, but where defaults should be
+// Configuration defaults
```
Comment (nandor-magyar) [inline golang/internal/config/config.go:7]: I would add a simple, inline, textual flow diagram.
```
//
// application startup -> load Configuration (read environment variables) -> unwrap variables (custom type)
//
// the last step is a public SetValue function defined on the custom type
// similar to json/yaml deserializing
```

Or similar, wherever you see fit.

**Review by nandor-magyar — APPROVED** [review]

File: golang/internal/config/config.go [hunk 2]
```diff
@@ -4,11 +4,18 @@ import (
...
+//
+// Convention is we have all the options here, but where defaults should be
 // different we handle it on application level, every time we need a parameter
 // we call into a function in a package which calls here, then we populate the
-// options which should be different if not defined otherwise
+// options which should be different if not defined otherwise.
+//
+// Using custom types allows to set the value from the environment variable
+// you need to impement the Setter interface on the field level.
```
[the remark this answers is not in the document]
Comment (polaroi8d) [reply golang/internal/config/config.go:15]: EAGLE EYEs!!!

**Review by nandor-magyar — APPROVED** [review]

File: golang/pkg/dagent/utils/docker.go [hunk 3]
```diff
@@ -65,12 +65,13 @@ func GetContainerLogs(name string, skip, take uint) []string {
...
 	}
 
+	const BASE = 10
 	tail := skip + take
 
 	options := types.ContainerLogsOptions{
 		ShowStderr: true,
 		ShowStdout: true,
-		Tail:       strconv.FormatUint(uint64(tail), 10),
+		Tail:       strconv.FormatUint(uint64(tail), BASE),
```
[the remark this answers is not in the document]
Comment (polaroi8d) [reply golang/pkg/dagent/utils/docker.go:74]: I am a simple man, I see `lint-errors`, I fix. Can I merge or undo it?

**Review by robot9706 — APPROVED** [review]
LGTM 👀

**Pull request merged**